src/migration-wizard/steps/discovery/EnhancedDownloadButton.tsx

```
@@ -449,9 +517,13 @@ const generateHTMLTemplate = (chartData: any, inventory: any): string => {
                 new Chart(powerCtx, {
                     type: 'doughnut',
                     data: {
-                        labels: ${JSON.stringify(powerStateData.map((d: any) => d[0]))},
+                        labels: ${JSON.stringify(
+                          powerStateData.map((d: any) => d[0]),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)